modules/react/banner/lib/Banner.tsx

```diff
@@ -1,33 +1,66 @@
 import * as React from 'react';
 
-import {borderRadius, CSSProperties, space, type} from '@workday/canvas-kit-react/tokens';
+import {borderRadius} from '@workday/canvas-kit-react/tokens';
 import {createContainer, ExtractProps, focusRing, useIsRTL} from '@workday/canvas-kit-react/common';
 import {Flex} from '@workday/canvas-kit-react/layout';
 
-import {useBannerModel, useThemedPalette} from './hooks';
+import {useBannerModel} from './hooks';
 
 import {BannerIcon} from './BannerIcon';
 import {BannerLabel} from './BannerLabel';
 import {BannerActionText} from './BannerActionText';
-import {px2rem} from '@workday/canvas-kit-styling';
-import {system} from '@workday/canvas-tokens-web';
-
+import {createStencil, handleCsProp, px2rem} from '@workday/canvas-kit-styling';
+import {brand, system} from '@workday/canvas-tokens-web';
 export interface BannerProps extends ExtractProps<typeof Flex, never> {
   /**
    * Children of the Banner. Should contain a `<Banner.Label>` a <Banner.Icon> and an optional `<Banner.ActionText>`
    */
   children?: React.ReactNode;
 }
 
-const styles: CSSProperties = {
-  cursor: 'pointer',
-  transition: 'background-color 120ms',
-  outline: `${system.space.x1} solid transparent`,
-  '&:focus-visible, &.focus': {
-    outline: `${px2rem(4)} double transparent`,
-    ...focusRing({separation: 2}),
+export const bannerStencil = createStencil({
+  base: {
+    ...system.type.subtext.large,
+    fontWeight: system.fontWeight.medium,
+    padding: `${system.space.x2} ${system.space.x4}`,
+    border: '0',
+    display: 'flex',
```

Comment:
Since you're already using `display: 'flex'` here, we can removed the use of the `Flex` component later.